docs/specification.md

```diff
@@ -21,6 +21,19 @@ These specifications were assembled with the following references:
 
 - The token contract must store tokens of a single type.
 
+# Questions
+
+1. How many addresses can be assigned to each role?
```

Comment:
Well, I think that FA2 spec clearly says that each role has exactly one address associated to it, otherwise we face a lot of consensus (for instance multiple master minters that make a cyclic dependency to one another allowing for generation of infinite amount of coins) problems that aren't mentioned anywhere

Reply:
FA2 does not know anything about roles like "master minter". Where does it say that each has exactly one address?

Reply:
Because it leads to a whole bunch of problems that I described above and that in any case should be mentioned in FA2. By looking at its example implementation I can say that it was stated to use only one.

Reply:
@gromakovsky I think it is coming from https://github.com/centrehq/centre-tokens/blob/master/doc/tokendesign.md

If you check the section "Reassigning Roles", the content there seem to imply that each role, except that of 'minters' can be assigned to only one address at a time.

Reply:
Yes, but
1. "seem to imply" does not make me 100% sure, it's better to ask explicitly.
2. Maybe they can be assigned to at most one address rather than exactly one address. E. g. maybe it's ok to have no pausers.